README.md

```diff
@@ -1,5 +1,6 @@
 
-# Pa11y CI
+Pa11y CI
+========
```

Comment:
Agh sorry this is an inconsistency in Pa11y – I'd rather move to the `#` headings than the underlined ones I think as the underlines get inconsistent. Sorry

Reply:
👀

Reply:
Dammit! I should've stuck to what I like... much prefer the #s myself